website/docs/r/custom_ip_prefix.html.markdown

```diff
@@ -0,0 +1,85 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_custom_ip_prefix"
+description: |-
+  Manages a Custom Ip Prefix.
+
+---
+
+# azurerm_custom_ip_prefix
+
+Manages a Custom Ip Prefix.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_custom_ip_prefix" "example" {
+  name                          = "example-CustomIpPrefix"
+  location                      = azurerm_resource_group.example.location
+  resource_group_name           = azurerm_resource_group.example.name
+  cidr                          = "0.0.0.0/24"
+  action                        = "Provision"
+  roa_expiration_date           = "20991212"
+  wan_validation_signed_message = "singed message for WAN validation"
```

Comment:
"singed", shouldn't that be "signed" ?

Reply:
@GurliGebis thanks for your feedback. Fixed.

Reply:
You're welcome 🙂